File: Day4/C/Ques14.c
#include<stdio.h>
int main()
{
    int a=0,b=1,c,n;
    printf("Enter value of n:");
    scanf("%d",&n);
    for(int i=0;i<n-1;i++){
        c=a+b;
        a=b;
        b=c;
    }
    printf(" %d th fibonaaci series terms is %d",n,a);
    return 0;
    
}

File: Day4/Python/Ques14.py
a,b=0,1
n=(int(input("Enter number of terms:")))
print("Fibonaaci series upto",n,"terms")
for i in range (0,n-1):
    temp=b
    b=a+b
    a=temp
print(n,"th Fibonaaci term is:",a)